src/views/Property/PropertyView.jsx

```diff
@@ -202,9 +190,20 @@ const Property = () => {
       });
   }
 
-  const removeTenant = (id) => {
+  const removeTenant = (tenant) => {
+    setTenantToRemove(tenant);
+    setConfirmTenantRemoval(true);
+  }
 
-    getProperty()
+  const handleTenantConfirmButton = () => {
+    const leaseId = tenantToRemove.lease.id;
+    axios
+      .delete(`${process.env.REACT_APP_PROXY}/api/lease/${leaseId}`)
+      .then(res => getProperty())
```

Comment:
We can simplify this by moving the actions within the following 3 then statements up into this one, since none of them rely on the getProperty method's result

Reply:
Good eye, I wasn't thinking!